src/test/java/com/solodev/clickergamebackend/ClickerGameBackEndApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.solodev.clickergamebackend;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
```

Comment:
If you're not doing any integration testing right now, then it's probably best to remove this.
Later, you can add it back